### PR TITLE
Add A7b2

### PR DESCRIPTION
**Pull request by CarterKoala**
Ensures the time is written down for a DNF

### CODE REVIEW

File: wca-guidelines.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # <wca-title>WCA Guidelines
 
-<version>Version: January 1, 2024
+<version>Version: January 1, 2025
```
Comment (Nevseros) [inline wca-guidelines.md]:
```suggestion
<version>Version: January 1, 2024
```

File: wca-regulations.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # <wca-title>WCA Regulations
 
-<version>Version: January 1, 2024
+<version>Version: January 1, 2025
```
Comment (Nevseros) [inline wca-regulations.md]:
```suggestion
<version>Version: January 1, 2024
```

File: wca-regulations.md [hunk 3]
```diff
@@ -448,6 +448,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
         - A7a1) The judge should indicate to the competitor that the puzzle is solved, or that a penalty will be applied.
     - A7b) The judge records the result on a score sheet and signs next to the result to acknowledge that the result is correct, complete, properly formatted, and clearly readable.
         - A7b1) If penalties are assigned, the judge records the original recorded time displayed on the timer, along with any penalties. The format should be "T + X = F", where T represents the time displayed on the timer (the "original recorded time"), X represents a sum of time penalties during the attempt, and F represents the final result (e.g. 17.65 + 4 = 21.65).
```
Comment (ohexter) [inline wca-regulations.md]:
```suggestion
        - A7b1) If time penalties are assigned, the judge records the original recorded time displayed on the timer, along with any penalties. The format should be "T + X = F", where T represents the time displayed on the timer (the "original recorded time"), X represents a sum of time penalties during the attempt, and F represents the final result (e.g. 17.65 + 4 = 21.65).
```
To better distinguish from DNF in addition.

Comment (CarterKoala) [reply wca-regulations.md]: Ah, I was trying to figure out how to best include that :)

File: wca-regulations.md [hunk 4]
```diff
@@ -447,7 +447,8 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
...
         - A7a1) The judge should indicate to the competitor that the puzzle is solved, or that a penalty will be applied.
     - A7b) The judge records the result on a score sheet and signs next to the result to acknowledge that the result is correct, complete, properly formatted, and clearly readable.
-        - A7b1) If penalties are assigned, the judge records the original recorded time displayed on the timer, along with any penalties. The format should be "T + X = F", where T represents the time displayed on the timer (the "original recorded time"), X represents a sum of time penalties during the attempt, and F represents the final result (e.g. 17.65 + 4 = 21.65).
+        - A7b1) If time penalties are assigned, the judge records the original recorded time displayed on the timer, along with any penalties. The format should be "T + X = F", where T represents the time displayed on the timer (the "original recorded time"), X represents a sum of time penalties during the attempt, and F represents the final result (e.g. 17.65 + 4 = 21.65).
+        - A7b2) If a DNF is assigned, the judge records the original time displayed on the timer, as well as "DNF".
```
Comment (Jobarion) [inline wca-regulations.md]: What should be recorded if both a DNF and a time penalty would have been applicable for different reasons?

Comment (CarterKoala) [reply wca-regulations.md]: Interesting point. I don't think this has ever been encountered before, but certainly could come into play if a competitor starts the timer with their palms but DNFs the solve. A1a2+ already uses "original recorded time", so I don't really have a problem with keeping it this way.
(Also, I neglected to realize that the wording could just be stolen from there :p)

**Pull request merged**